index.js

```diff
@@ -77,41 +77,63 @@ function getScopedCid(scope, apiKey, callback) {
     if (!token || isStatusToken(token)) {
       persistToken(TokenStatus.RETRIEVING, TIMEOUT);
     }
-    fetchCid(GOOGLE_API_URL + apiKey, scope, token, cb)
+    fetchCid(apiKey, scope, token, cb)
   });
 }
 
 /**
  * Fetches CID remotely
  *
- * @param {string} url
+ * @param {string} apiKey
  * @param {string} scope
  * @param {string=} token
  * @param {function()} callback
  */
-function fetchCid(url, scope, token, callback) {
+function fetchCid(apiKey, scope, token, callback) {
   var payload = {'originScope': scope};
   if (token) {
     payload['securityToken'] = token;
   }
+  const url = GOOGLE_API_URL + apiKey;
   fetchJson(url, payload, TIMEOUT, function (err, res) {
     if (err) {
```

Comment:
why not remove `if (err)` also?

Reply:
Was trying to check `error` first thing first. But I agree should remove `if (err) ` as well. done